### PR TITLE
Share button

### PR DESCRIPTION
**Pull request by NghiaNTT**

### CODE REVIEW

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -14,3 +14,4 @@
 //= require jquery_ujs
 //= require bootstrap
 //= require_tree .
+//= require social-share-button
```
Comment (nguyenhad) [inline app/assets/javascripts/application.js]: remove thí file

File: app/assets/stylesheets/application.css [hunk 2]
```diff
@@ -12,4 +12,5 @@
  *
  *= require_tree .
  *= require_self
+ *= require social-share-button
  */
```
Comment (nguyenhad) [inline app/assets/stylesheets/application.css]: remove this file

**Comment by nguyenhad** [comment]
ok

**Comment by hieunk58** [comment]
OK

**Pull request merged**